docs-source/content/faq/configmaps.md

```diff
@@ -0,0 +1,33 @@
+> I need to provide an instance with access to a Config Map
```

Comment:
Also, the problem statements in both FAQs should end with a period. I see that we missed that detail in the Cannot Pull Image FAQ; that should be fixed, not propagated.